js/coinex.js

```diff
@@ -2639,6 +2639,101 @@ module.exports = class coinex extends Exchange {
+        //         "data": {
+        //             "BTCUSD": [
+        //                 ["500001", "100", "0.005"],
+        //                 ["1000001", "50", "0.01"],
+        //                 ["2000001", "30", "0.015"],
+        //                 ["5000001", "20", "0.02"],
+        //                 ["10000001", "15", "0.025"],
+        //                 ["20000001", "10", "0.03"]
+        //             ],
+        //             ...
+        //         },
+        //         "message": "OK"
+        //     }
+        //
+        const data = this.safeValue (response, 'data', {});
+        return this.parseLeverageTiers (data, symbols, undefined);
+    }
+
+    async fetchMarketLeverageTiers (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (!market['contract']) {
+            throw new BadRequest (this.id + ' fetchMarketLeverageTiers() symbol supports contract markets only');
+        }
+        const response = await this.perpetualPublicGetMarketLimitConfig (params);
+        //
+        //     {
+        //         "code": 0,
+        //         "data": {
+        //             "BTCUSD": [
+        //                 ["500001", "100", "0.005"],
+        //                 ["1000001", "50", "0.01"],
+        //                 ["2000001", "30", "0.015"],
+        //                 ["5000001", "20", "0.02"],
+        //                 ["10000001", "15", "0.025"],
+        //                 ["20000001", "10", "0.03"]
+        //             ],
+        //             ...
+        //         },
+        //         "message": "OK"
+        //     }
+        //
+        const data = this.safeValue (response, 'data', {});
+        const tiers = this.parseLeverageTiers (data, [ symbol ], market['id']);
+        return this.safeValue (tiers, symbol);
+    }
+
+    parseLeverageTiers (response, symbols, marketIdKey) {
+        //
+        //     {
+        //         "BTCUSD": [
+        //             ["500001", "100", "0.005"],
+        //             ["1000001", "50", "0.01"],
+        //             ["2000001", "30", "0.015"],
+        //             ["5000001", "20", "0.02"],
+        //             ["10000001", "15", "0.025"],
+        //             ["20000001", "10", "0.03"]
+        //         ],
+        //         ...
+        //     }
+        //
+        const result = {};
+        const currencyIds = Object.keys (response);
+        for (let i = 0; i < currencyIds.length; i++) {
+            const currencyId = currencyIds[i];
+            const ladder = response[currencyId];
+            const tiers = [];
+            const symbol = this.safeSymbol (currencyId);
+            for (let j = 0; j < ladder.length; j++) {
+                const bracket = ladder[j];
+                const leverage = this.safeString (bracket, 1);
+                const maxNotional = this.safeString (bracket, 0);
+                const previousBracket = ladder[j - 1];
```

Comment:
You can get rid of this if you move `minNotional` like I mentioned in other comments